documentation/src/main/asciidoc/topics/proc_server_resp_differences.adoc

```diff
@@ -0,0 +1,33 @@
+[id='resp-differences_{context}']
+= {brandname} guarantees
```

Comment:
Perhaps the title could be something like "Differences between {brandname}  and Redis implementations"